source/YTEPs/YTEP-0036.rst

```diff
@@ -0,0 +1,248 @@
+
+.. code-block:: yaml
+
+  calling_function_name:
+    test_name: hash
+    test_parameter1_value1: value
+    test_parameter2_value1: value
+
+This produces small, human-readable text files that can be easily packaged with the main code base. This makes running and managing the tests simpler.
+
+The saving, loading, and comparison of the generated hashes has been implemented in a new convenience function ``handle_hashes(save_dir, save_file, hashes, answer_store)`` in ``yt/utilities/answer_testing/utils.py``.
+
+The ``save_dir`` argument is the path to the directory where the answers are either to be saved or loaded from, the ``save_file`` argument specifies the name of the answer file to either save to or load from, the ``hashes`` are the newly generated data to be either saved or loaded, and the ``answer_store`` argument specifies whether or not the passed data should be saved or compared.
+
+If ``answer_store`` is ``True`` then the passed ``hashes`` should be written to the specified yaml file. If ``answer_store`` is ``False`` then the passed ``hashes`` should be compared to the hashes loaded from the specified answer file.
+
+Running the Tests
+^^^^^^^^^^^^^^^^^
+
+Unit Tests
+""""""""""
+
+The goal was to keep the process as similar to the previous implementation as possible. Previously, unit tests could be run from the command line in the ``$YT_GIT`` directory (where this refers to the root of the yt repo) as
+
+.. code-block:: bash
+
+  $ nosetests
+
+Similarly, now they can be run with
+
+.. code-block:: bash
+
+  $ pytest
+
+Should one desire to disable pytest's default capturing of ``stdout``, the ``-s`` option can be invoked. If one would like a verbose description of the tests being run as well as their status (and a detailed report in the event of a failure), one can enable the ``-v`` flag.
+
+To run a specific test or group of tests, one can either pass in the path to the module containing the tests
+
+.. code-block:: bash
+
+  $ pytest -s -v /path/to/test/module.py
+
+or use pytest's powerful ``-k`` flag, which enables test selection by name. For instance, to run all of the tests contained in a single class, one would do:
+
+.. code-block:: bash
+
+  $ pytest -s -v -k "TestClass"
+
+and pytest will collect every test and then ignore all of the tests not contained in the ``TestClass`` class. To run only a specific method within a given class, one would do:
+
+.. code-block:: bash
+
+  $ pytest -s -v -k "TestClass and test_method"
+
+See `this link <https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests>`_ for more on pytest's selection capabilities and options.
+
+Answer Tests
+""""""""""""
+
+To run the answer tests previously, one had to first tell yt where the test data used in the answer tests was located via
+
+.. code-block:: bash
+
+  $ yt config set yt test_data_dir /path/to/yt-data
+
+This remains true here, as well.
```

Comment:
Not _exactly_ true anymore. Since https://github.com/yt-project/yt/pull/2981, it's now possible to have _local_ config files for yt. A side effect is that one can have as many data storage files as yt clones on a single machine.
However, the command line `yt` tool still only affects the global config file _unless specified otherwise_.
There no need to get into the details here but maybe a footnote could help.